### ISSUE TITLE
Bug: `no-unsafe-argument` missing placeholders in violation message

### ISSUE THREAD

**Issue by bmish** [typescript-eslint/typescript-eslint#5461]
### Before You File a Bug Report Please Confirm You Have Done The Following...

- [X] I have tried restarting my IDE and the issue persists.
- [X] I have updated to the latest version of the packages.
- [X] I have [searched for related issues](https://github.com/typescript-eslint/typescript-eslint/issues?q=is%3Aissue+label%3Abug+label%3A%22package%3A+eslint-plugin%22) and found none that matched my issue.
- [X] I have [read the FAQ](https://typescript-eslint.io/docs/linting/troubleshooting) and my problem is not listed.

### Playground Link

https://typescript-eslint.io/play/#ts=4.7.2&sourceType=module&code=CYUwxgNghgTiAEAzArgOzAFwJYHtVJxwApYBzARgC54BnDGLVUgGnjICZrVkBbAIxAwAlNQBuOLMADcAKBlg8deAA94AXngBtAORRtrcmxptUATwC6R+AtR1ZiQkQB0L5UKlA&eslintrc=N4KABGBEBOCuA2BTAzpAXGUEKQAIBcBPABxQGNoBLY-AWhXkoDt8B6Jge1tieQEMAZolp9oAc1gBbRC3RRE0aB2iRwYAL4h1QA&tsconfig=N4KABGBEDGD2C2AHAlgGwKYCcDyiAuysAdgM6QBcYoEEkJemy0eAcgK6qoDCAFutAGsylBm3TgwAXxCSgA

### Repro Code

```typescript
declare function foo(arg1: string, arg2: number): void;

const x = ['a', 1 as any] as const;
foo(...x);
```

### ESLint Config

```javascript
module.exports = {
  parser: "@typescript-eslint/parser",
  rules: {
    "@typescript-eslint/no-unsafe-argument": "error"
  },
};
```

### tsconfig

```jsonc
{
  "compilerOptions": {
    // ...
  }
}
```

### Expected Result

Displays violation message with missing placeholders:

```
Unsafe spread of a tuple type. The {{index}} element is of type `any` and is assigned to a parameter of type `{{reciever}}`. 4:5 - 4:9
```

### Actual Result

Displays a violation message without missing placeholders:

```
Unsafe spread of a tuple type. The argument is of type `any` and is assigned to a parameter of type `number`. 4:5 - 4:9
```

### Versions

| package                                | version |
| -------------------------------------- | ------- |
| `@typescript-eslint/eslint-plugin`     | `5.33.0` |
| `@typescript-eslint/parser`            | `5.33.0` |
| `TypeScript`                           | `4.7.4` |
| `ESLint`                               | `8.21.0` |
| `node`                                 | `18.170` |
